### PR TITLE
Adjust warnings

### PR DESCRIPTION
**Pull request by srliao**
This PR:
- adjusts warning message to make it more clear that they're ok to ignore
- removes warning sign from embeds as they are often misleading

### CODE REVIEW

File: ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx [hunk 1]
```diff
@@ -155,17 +148,15 @@ const StaminaWarning = ({ data }: WarningProps) => {
...
-        title="Insufficient Stamina"
+        title="Delay in Dash - Insufficient Stamina"
         intent={Intent.WARNING}
         show={visible}
         onDismiss={() => setShow(false)}>
       <p>
-        An abnormal amount of iterations failed to execute an action because the character did not
-        have enough stamina. Consider updating the config to better manage stamina, as no actions
-        are performed during failures.
+        Some iterations delayed executing dash due to insufficient stamina. This causes the active character to idle until enough stamina regnerated. Consider updating the config if the downtime below is undesired.
```
Comment (soloxcx) [inline ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx:156]: This section is about charged attacks, or any action that consumes stamina. Also fixing typos:

> Some iterations delayed executing a stamina-draining action due to insufficient stamina. This causes the active character to idle until enough stamina is regenerated. Consider updating the config if the downtime below is undesired.

File: ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx [hunk 2]
```diff
@@ -178,17 +169,15 @@ const DashWarning = ({ data }: WarningProps) => {
...
-        title="Unable to Use Character Dash (Dash on CD)"
+        title="Delay in Dash - Dash on CD"
         intent={Intent.WARNING}
         show={visible}
         onDismiss={() => setShow(false)}>
       <p>
-        An abnormal amount of iterations failed to execute a dash because the dash was on cooldown.
-        Consider updating the config to better account for dash cooldowns, as no actions are
-        performed during failures.
+        Some iterations delayed executing dash due to its cooldown. This causes the active character to idle until enough stamina regnerated. Consider updating the config if the downtime below is undesired.
```
Comment (soloxcx) [inline ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx:177]: > Some iterations delayed executing one or more dashes due to its cooldown. This causes the active character to idle until enough stamina is regenerated. Consider updating the config if the downtime below is undesired.

Comment (srliao) [reply ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx]: fixed

File: ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx [hunk 3]
```diff
@@ -73,8 +73,7 @@ const PositionOverlapWarning = ({ data }: WarningProps) => {
         show={visible}
         onDismiss={() => setShow(false)}>
       <p>
-        {"Target position's overlap in at least one iteration. This may result in inaccurate "
-          + "simulations. Update positions to avoid any overlaps."}
+        {"Target position's overlap in at least on iteration. Confirm if this is intended and update positions to avoid overlaps as necessary. Overlapping positions may result in inaccurate simulations."}
```
Comment (k0l11) [inline ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx:76]: on -> one

**Pull request merged**